lib/protocol/Writer.js

```diff
@@ -357,6 +358,8 @@ Writer.prototype.pushLob = function pushLob(buffer, value) {
     stream.push(null);
   } else if (value instanceof Readable) {
     stream = value;
+  } else if (value instanceof Lob) {
+    stream = value.createReadStream();
```

Comment:
Could we check the validity of the Lob here?
If the Lob is in an invalid state report that error to the user.
(when in an invalid state value.createReadStream() would return null - the check is on the variable _running)